features/providers/non_passported_journey.feature

```diff
@@ -65,9 +66,28 @@ Feature: Non-passported applicant journeys
     And I click 'Save and continue'
 
     Then I should be on the 'income_summary' page showing "Sort your client's income into categories"
+    And the following sections should exist:
+      | tag | section |
+      | h2  | Benefits |
+      | h2  | Disregarded benefits |
+      | h2  | Financial help from friends or family |
+
+    And the following sections should not exist:
+      | tag | section |
+      | h2  | Housing benefits |
```

Comment:
not a blocker but for this page I would use be tempted to use a more general

And I should not see 'Housing benefit'